selectors/general.js

```diff
@@ -21,5 +21,5 @@ export function getBasePath(state) {
         return new URL(config.SiteURL).pathname;
     }
 
-    return window.basename || window.location.origin;
+    return window.basename || '/';
```

Comment:
Not sure why I ever made this default to `.origin` -- that includes a hostname and port, which is useless as a basename.